chalicelib/es_connection.py

```diff
@@ -181,7 +182,7 @@ def get_result_history(self, prefix, start, limit):
         search.update_from_dict(doc)
         return self.search(search)
 
-    def get_main_page_checks(self, primary=True):
+    def get_main_page_checks(self, checks=None, primary=True):
```

Comment:
Please add a test to `test_es_connection` that leverages `get_main_page` with checks argument